ansys/content.jsonc

```
@@ -10,6 +10,7 @@
     "long_desc": [
         "Ansys suite of computational simulation products are available to DesignSafe users, including the structural analysis and computational fluid dynamics modules. Most simulations are performed using the Ansys Workbench system.",
         "TACC's Ansys license is available only for non-commercial, academic use. To access Ansys please <a href=\"https://www.designsafe-ci.org/help/new-ticket/\">submit a ticket</a> and provide your institutional affiliation and a brief statement confirming that you will use Ansys only for non-commercial, academic purposes."
+        "Note: Ansys is only accessible by logging to TACC's supercomputers via <a href=\"https://tap.tacc.utexas.edu/\">TACC Analysis Portal (TAP)</a> or Secure Shell (SSH). More details on how to use Ansys on TACC can be found by clicking the User Guide button below."
     ],
     "is_developed_by_simcenter": false,
     "user_guide_desc": "Read step-by-step instructions on getting started in the User Guide.",
```

Ah, the other bit:

```suggestion
    "user_guide_desc": "Read step-by-step instructions on getting started in TACC's User Guide.",
```

I added "on TACC" to the user guide title as well (which is not available int he JSON).